tests/performance/load/groups/alerts.js

```diff
@@ -0,0 +1,40 @@
+import {Client, StatusOK} from 'k6/net/grpc';
+import {check, group} from 'k6';
+
+const GRPC_PORT = ":443";
+
+const client = new Client();
+
+client.load(['../../../proto', '../../../third_party/googleapis'], 'api/v1/alert_service.proto');
```

Comment:
Is there somehow that we can avoid having all of these `../../../`?

Reply:
Or pass root path in env